src/ucs/datastruct/bitmap.h

```diff
@@ -229,6 +231,20 @@ typedef uint64_t ucs_bitmap_word_t;
 #define ucs_bitmap_t(_length) ucs_bitmap_##_length##_t
 
 
+/**
+ * A boolean wrapper for the ucs_assert function, to insert it to conditionals
+ * 
+ * @param condition Assert this condition
+ * 
+ * @return true
+ */
+static inline bool _ucs_bitmap_assert(bool condition)
+{
+    ucs_assert(condition);
+    return true;
+}
```

Comment:
maybe just create macros with `({  assert(); _expr; })`

Reply:
How would that solve it? It won't fall on `assert()` if `_expr` is false, but that's what we'd like to happen.

Reply:
no need to create assert which returns a value, can just prepend assert statement inside the macro